### PR TITLE
Affordable Connectivity Program

### PR DESCRIPTION
**Pull request by MaxGhenis**
# New variable: `acp`

- [x] Label field added
- [x] Documentation field added
- [x] Unit field added
- [x] Default value field added if relevant
- [x] Variable name follows conventions
- [x] Unit test(s) added
- [x] Integration test(s) added if relevant
- [x] Issues this PR fixes linked

## What's changed

Adds the Affordable Connectivity Program, which essentially replaced the Emergency Broadband Benefit with a similar structure and different parameters.

While the [Emergency Broadband Benefit is in the CFR](https://www.law.cornell.edu/cfr/text/47/part-54/subpart-P), I didn't see the ACP in there. For now, it points to FCC websites, but I'll keep looking.

### CODE REVIEW

**Comment by nikhilwoodruff** [comment]
> @nikhilwoodruff everything seems to work fine, but in the notebook I'm getting:
> 
> > WARNING:root:Parameter fcc.acp.amount has a child True that is not in the possible values of is_on_tribal_land, ignoring.
> 
> Do nested parameters work with bool breakdowns?

Just tested - FYI this is a warning, and nothing's broken (yet). It just means that the auto-breakdown-filling function is adding children named `true` and `false`, which aren't used because they're incorrectly named.

**Review by nikhilwoodruff — APPROVED** [review]

**Comment by MaxGhenis** [comment]
@nikhilwoodruff could you please review my new school meal subsidy changes? If it looks OK, this is ready to merge.

File: openfisca_us/variables/fcc/acp/acp.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+
+class acp(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "Affordable Connectivity Program amount"
+    documentation = "Affordable Connectivity Program amount"
+    definition_period = YEAR
+    unit = USD
+    reference = "https://www.fcc.gov/acp"
```
Comment (nikhilwoodruff) [inline openfisca_us/variables/fcc/acp/acp.py:11]: Alternatively could reference the U.S. Code source: http://uscode.house.gov/view.xhtml?req=granuleid:USC-prelim-title47-section1752&num=0&edition=prelim

File: openfisca_us/variables/fcc/acp/is_acp_eligible.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+from openfisca_us.model_api import *
+
+
+class is_acp_eligible(Variable):
+    value_type = bool
+    entity = SPMUnit
+    label = "Eligible for Affordable Connectivity Program"
+    documentation = "Eligible for Affordable Connectivity Program"
+    definition_period = YEAR
+
```
Comment (nikhilwoodruff) [inline openfisca_us/variables/fcc/acp/is_acp_eligible.py:10]:
```suggestion
reference = "47 U.S.C §1752(a)(6)"
```

Or the [link](http://uscode.house.gov/view.xhtml?req=granuleid:USC-prelim-title47-section1752&num=0&edition=prelim), though I don't think we can link to specific sections?

**Review by nikhilwoodruff — APPROVED** [review]

File: openfisca_us/variables/fcc/acp/acp.py (hunk 1 above)
Comment (MaxGhenis) [reply openfisca_us/variables/fcc/acp/acp.py:11]: thanks!

File: openfisca_us/variables/fcc/acp/is_acp_eligible.py (hunk 2 above)
Comment (MaxGhenis) [reply openfisca_us/variables/fcc/acp/is_acp_eligible.py:10]: Filed https://github.com/PolicyEngine/openfisca-us/issues/581 to come back to this once it's in the Cornell site

**Pull request merged**